tests/mypy_test.py

```diff
@@ -531,7 +533,7 @@ def test_typeshed(code: int, args: TestConfig, tempdir: Path) -> TestResults:
 
 
 def main() -> None:
-    args = parser.parse_args(namespace=CommandLineArgs())
+    args = parser.parse_args(namespace=CommandLineArgs)
```

Comment:
What is the motivation for this change? Makes more sense to pass an instance here.

Reply:
Indeed this is no longer necessary with `@dataclass(init=False)`